src/Tribe/Main.php

```diff
@@ -3807,17 +3807,25 @@ public function isEvent( $event ) {
 		 * @return bool Is it a venue?
 		 */
 		public function isVenue( $postId = null ) {
+			static $is_venue = [];
+
 			if ( $postId === null || ! is_numeric( $postId ) ) {
 				global $post;
 				if ( isset( $post->ID ) ) {
 					$postId = $post->ID;
 				}
 			}
+
+			// Return if we've already fetched this info.
+			if ( isset( $is_venue[ $postId ] ) ) {
+				return $is_venue[ $postId ];
+			}
+
 			if ( isset( $postId ) && get_post_field( 'post_type', $postId ) == Tribe__Events__Venue::POSTTYPE ) {
-				return true;
+				return $is_venue[ $postId ] = true;
 			}
 
-			return false;
+			return $is_venue[ $postId ] = false;
```

Comment:
* :no_entry_sign: Variable "$postId" is not in valid snake_case format, try "$post_id"
* :no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 94f19ca120be81bf9bec041805517e78</sub>

Reply:
Ignoring.